config/application.rb

```diff
@@ -17,6 +19,10 @@ class Application < Rails::Application
       g.fixture_replacement :factory_bot_rails, dir: 'spec/factories'
     end
 
+    Raven.configure do |config|
```

Comment:
wrong place add a separate initializer
url should be in .env file not shown openly

Reply:
done